api-2.2/src/main/java/org/openmrs/module/coreapps/htmlformentry/EncounterDiagnosesElement.java

```diff
@@ -121,7 +122,6 @@ public String generateHtml(FormEntryContext context) {
             try {
                 Map<String, Object> fragmentConfig = new HashMap<String, Object>();
                 fragmentConfig.put("formFieldName", "encounterDiagnoses");
-                fragmentConfig.put("existingDiagnoses", existingDiagnoses);
```

Comment:
Why did you remove the above?

Reply:
@dkayiwa i had removed i realised i had removed it while reviewing and i put it back